playbooks/tasks/set-deployment-facts.yml

```
@@ -22,8 +22,8 @@
       packit-dashboard: "{{ with_dashboard }}"
       pushgateway: "{{ with_pushgateway }}"
       nginx: "{{ with_pushgateway }}"
-      redis: "{{ with_redis }}"
-      redict: "{{ with_redict }}"
+      redis: "{{ with_kv_database and kv_database == 'redis' }}"
+      redict: "{{ with_kv_database and kv_database == 'redict' }}"
```

```suggestion
      redict: "{{ with_kv_database and kv_database == 'redict' }}"
      valkey: "{{ with_kv_database and kv_database == 'valkey' }}"
```
Not sure why we need this facts but isn't `valkey` missing here?

---

Oh, right, must've missed it.

This builds up the `optional_deploymentconfigs`, from that we build the `deploymentconfigs` and finally it's checked here

https://github.com/packit/deployment/blob/2892b42d7cfb05d36a51ad239f73927fc0ef2b58/playbooks/deploy.yml#L440-L446

that they're running. It's a bit flaky though, cause it doesn't work properly if you don't run `oc project ‹packit namespace›` beforehand in the terminal.